src/d3d11/d3d11_device.cpp

```diff
@@ -363,7 +363,9 @@ namespace dxvk {
     if (hr != S_OK)
       return hr;
     
-    *ppSRView = view;
```

Comment:
Why go through all the work of creating the SRV only to destroy it immediately again?
Check the pointer at the beginning of the function and return early.

Judging by `CreateShaderResourceViewBase`, it should also return `S_FALSE`.